nextflow.config

```diff
@@ -50,10 +50,27 @@ params {
     validationShowHiddenParams       = false
     validate_params                  = true
 
-    //StarAMR options
+    // StarAMR options
+
+    // Databases
     pointfinder_database             = null
     plasmidfinder_database           = null
     mlst_scheme                      = null
+    minimum_contig_length            = null
```

Comment:
You are correct that it is not necessary to run the pipeline with these defaults set. But it will help out with documenting and showing users what parameters are being used (whether in nextflow help parameters or in IRIDA Next UI).

That is, with defaults set, the IRIDA Next UI will show them to people (e.g., like in mikrokondo below)

![image](https://github.com/phac-nml/staramrnf/assets/200517/5ed0e527-77a8-400c-bfc8-5ebd1fa33fea)

The warning might be because you need defaults in both `nextflow.config` and `nextflow_schema.json`, which can be done with `nf-core schema build`